cmd/opampsupervisor/supervisor/supervisor.go

```diff
@@ -117,9 +116,7 @@ type Supervisor struct {
 
 	startedAt time.Time
 
-	healthCheckTicker  *backoff.Ticker
-	healthChecker      *healthchecker.HTTPHealthChecker
-	lastHealthCheckErr error
+	healthChecker *healthchecker.HTTPHealthChecker
```

Comment:
Do we still need this?

Reply:
There will be lots to remove, I'll clean it all up when I remove the health check extension stuff in the next PR.

I wanted to focus this PR only on switching to the agent's health message and tidying up any lint issues from that switch.

Reply:
Got it, makes sense.